static/js/compose_validate.js

```diff
@@ -173,6 +173,9 @@ export function clear_topic_resolved_warning() {
 }
 
 export function warn_if_topic_resolved(topic_changed) {
+    if (compose_state.get_message_type() !== "stream") {
+        return;
+    }
```

Comment:
This might be better as a prep commit.

That prep commit might want to add a `compose_state.recipient_has_topics` helper that just does `return compose_state.get_message_type() !== "stream";` and calling that, just to make it super readable why we're doing this, and that it's valid to access the topic just after.

Reply:
Ah I like that, thanks :)